confidant/models/credential.py

```diff
@@ -184,11 +184,11 @@ def next_rotation_date(self):
         # If a credential has never been rotated or been decrypted,
         # immediately rotate
         if self.last_rotation_date is None:
-            return datetime.utcnow()
+            return datetime.now()
```

Comment:
now() has the timezone so we don't do subtraction between tz aware and tz unaware datetimes